src/modules/sensors/sensor_params.c

```diff
@@ -227,6 +227,31 @@ PARAM_DEFINE_INT32(SENS_EN_THERMAL, -1);
 */
 PARAM_DEFINE_FLOAT(IMU_GYRO_CUTOFF, 30.0f);
 
+/**
+* Enable computation of angular acceleration by gyro differentiation.
+*
+* @reboot_required true
+*
+* @boolean
+* @group Sensors
+*/
+PARAM_DEFINE_INT32(IMU_DGYRO_EN, 0);
```

Comment:
I would briefly check top on an F4, but it's probably fine to leave on.

Reply:
Here is top with IMU_DGYRO_EN=0
```
[2019-11-21T18:42:39.735Z] nsh> 
[2019-11-21T18:42:39.735Z] nsh> top once
[2019-11-21T18:42:39.735Z] 
[2019-11-21T18:42:39.735Z]  PID COMMAND                   CPU(ms) CPU(%)  USED/STACK PRIO(BASE) STATE FD
[2019-11-21T18:42:39.735Z]    0 Idle Task                  104771 97.800   244/  512   0 (  0)  READY  3
[2019-11-21T18:42:39.735Z]    1 hpwork                          0  0.000   344/ 1260 249 (249)  w:sig  3
[2019-11-21T18:42:39.735Z]    2 lpwork                          3  0.000   944/ 1516  50 ( 50)  w:sig  3
[2019-11-21T18:42:39.735Z]    3 init                         1437  0.000  1304/ 2604 100 (100)  w:sem  3
[2019-11-21T18:42:39.735Z]    4 wq:manager                      0  0.000   376/ 1252 243 (243)  w:sem  3
[2019-11-21T18:42:39.735Z]   41 wq:att_pos_ctrl                 0  0.000   680/ 6596 244 (244)  w:sem  3
[2019-11-21T18:42:39.735Z]    8 wq:I2C2                        28  0.100   624/ 1396 245 (245)  w:sem  3
[2019-11-21T18:42:39.735Z]   22 px4io                         993  0.800   884/ 1484 240 (240)  w:sem  4
[2019-11-21T18:42:39.735Z]   27 wq:SPI1                       554  0.400   600/ 1396 253 (253)  w:sem  3
[2019-11-21T18:42:39.735Z]   29 wq:hp_default                 893  0.700   560/ 1500 243 (243)  w:sem  3
[2019-11-21T18:42:39.735Z]   39 sensors                       240  0.200  1344/ 1964 237 (237)  w:sem  5
[2019-11-21T18:42:39.735Z]   70 top                             0  0.000  1248/ 2020 248 (248)  RUN    3
[2019-11-21T18:42:39.735Z]   43 wq:rate_ctrl                    0  0.000   688/ 1596 255 (255)  w:sem  3
```
waiting CI for the same with IMU_DGYRO_EN=1

Reply:
With IMU_DGYRO_EN=1
```
[2019-11-22T18:31:47.429Z]  PID COMMAND                   CPU(ms) CPU(%)  USED/STACK PRIO(BASE) STATE FD
[2019-11-22T18:31:47.429Z]    0 Idle Task                   80986 97.800   244/  512   0 (  0)  READY  3
[2019-11-22T18:31:47.429Z]    1 hpwork                          0  0.000   344/ 1260 249 (249)  w:sig  3
[2019-11-22T18:31:47.429Z]    2 lpwork                          3  0.000   944/ 1516  50 ( 50)  w:sig  3
[2019-11-22T18:31:47.429Z]    3 init                         1437  0.000  1304/ 2604 100 (100)  w:sem  3
[2019-11-22T18:31:47.429Z]    4 wq:manager                      0  0.000   384/ 1252 243 (243)  w:sem  3
[2019-11-22T18:31:47.429Z]   41 wq:att_pos_ctrl                 0  0.000   680/ 6596 244 (244)  w:sem  3
[2019-11-22T18:31:47.429Z]    8 wq:I2C2                        24  0.000   624/ 1396 245 (245)  w:sem  3
[2019-11-22T18:31:47.429Z]   22 px4io                         813  0.700  1016/ 1484 240 (240)  w:sem  4
[2019-11-22T18:31:47.429Z]   27 wq:SPI1                       450  0.400   600/ 1396 253 (253)  w:sem  3
[2019-11-22T18:31:47.429Z]   29 wq:hp_default                 692  0.600   584/ 1500 243 (243)  w:sem  3
[2019-11-22T18:31:47.429Z]   39 sensors                       207  0.200  1328/ 1964 237 (237)  w:sem  5
[2019-11-22T18:31:47.429Z]   70 top                             0  0.000  1248/ 2020 248 (248)  RUN    3
[2019-11-22T18:31:47.429Z]   43 wq:rate_ctrl                    0  0.000   688/ 1596 255 (255)  w:sem  3
```

less CPU and less stack :thinking:

Reply:
Where are you looking? The sensors don't seem to be running.

For this PR this is what I see on fmu-v4.

![image](https://user-images.githubusercontent.com/84712/69461029-99a63d00-0d43-11ea-83ff-927b95da3f95.png)


http://ci.px4.io:8080/blue/organizations/jenkins/PX4_misc%2FFirmware-hardware/detail/pr-ratectrl_torque_accel_inertia/14/pipeline/667

Reply:
I was looking at px4_fmu-v2_default / test.

Reply:
px4_fmu-v4_default without IMU_DGYRO_EN
```
[2019-11-21T18:45:24.193Z]  PID COMMAND                   CPU(ms) CPU(%)  USED/STACK PRIO(BASE) STATE FD
[2019-11-21T18:45:24.193Z]    0 Idle Task                   50020 32.467   244/  512   0 (  0)  READY  3
[2019-11-21T18:45:24.193Z]    1 hpwork                        849  0.599   640/ 1260 249 (249)  w:sig  4
[2019-11-21T18:45:24.193Z]    2 lpwork                        110  0.099   704/ 1516  50 ( 50)  w:sig  8
[2019-11-21T18:45:24.193Z]    3 init                         1575  0.000  1864/ 2604 100 (100)  w:sem  3
[2019-11-21T18:45:24.193Z]    4 wq:manager                      0  0.000   376/ 1252 243 (243)  w:sem  4
[2019-11-21T18:45:24.193Z]  183 wq:att_pos_ctrl             17433 11.288  4768/ 6596 244 (244)  READY  4
[2019-11-21T18:45:24.193Z]   16 wq:hp_default                1630  1.098  1128/ 1500 243 (243)  w:sem  4
[2019-11-21T18:45:24.193Z]   18 dataman                         1  0.000   776/ 1204  90 ( 90)  w:sem  4
[2019-11-21T18:45:24.193Z]   21 wq:lp_default                  23  0.000   780/ 1700 205 (205)  w:sem  4
[2019-11-21T18:45:24.193Z]  143 wq:I2C1                       633  0.399   804/ 1396 246 (246)  w:sem  4
[2019-11-21T18:45:24.193Z]  152 wq:SPI2                       688  0.399   600/ 1396 252 (252)  w:sem  4
[2019-11-21T18:45:24.193Z]  156 wq:SPI1                     35950 23.176   992/ 1396 253 (253)  w:sem  4
[2019-11-21T18:45:24.193Z]  182 sensors                      5654  3.696  1308/ 1964 237 (237)  READY 10
[2019-11-21T18:45:24.193Z]  596 log_writer_file              1334  0.999   616/ 1164  60 ( 60)  w:sem  4
[2019-11-21T18:45:24.193Z]  187 wq:rate_ctrl                13639  8.691  1104/ 1596 255 (255)  w:sem  4
[2019-11-21T18:45:24.193Z]  189 commander                    1433  0.899  1592/ 3212 140 (140)  w:sig  6
[2019-11-21T18:45:24.193Z]  190 commander_low_prio              3  0.000   632/ 2996  50 ( 50)  w:sem  6
[2019-11-21T18:45:24.193Z]  200 mavlink_if0                  7442  4.695  1712/ 2572 100 (100)  w:sig  4
[2019-11-21T18:45:24.193Z]  201 mavlink_rcv_if0               557  0.299  2464/ 3972 175 (175)  w:sem  4
[2019-11-21T18:45:24.193Z]  204 mavlink_if1                  6415  4.095  1696/ 2540 100 (100)  w:sig  4
[2019-11-21T18:45:24.193Z]  205 mavlink_rcv_if1               615  0.299  2576/ 3972 175 (175)  w:sem  4
[2019-11-21T18:45:24.193Z]  273 gps                           376  0.199  1144/ 1540 208 (208)  w:sem  4
[2019-11-21T18:45:24.193Z]  314 mavlink_if2                  2514  1.298  1648/ 2484 100 (100)  w:sig  4
[2019-11-21T18:45:24.193Z]  319 mavlink_rcv_if2               634  0.399  2576/ 3972 175 (175)  READY  4
[2019-11-21T18:45:24.193Z]  523 navigator                     234  0.099   984/ 1764 105 (105)  w:sem  5
[2019-11-21T18:45:24.193Z]  709 top                             1  0.000  1248/ 2020 248 (248)  RUN    3
[2019-11-21T18:45:24.193Z]  586 logger                       3277  2.197  3088/ 3644 233 (233)  w:sem  4
```

http://ci.px4.io:8080/blue/organizations/jenkins/PX4_misc%2FFirmware-hardware/detail/pr-ratectrl_torque_accel_inertia/13/pipeline/601

Reply:
px4_fmu-v2_test is an old dedicated init that only ran the `tests` command. 
https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_test/init.d/rcS

We should probably kill it.